modules/default-requirements-secured-cluster-services.adoc

```diff
@@ -14,6 +14,8 @@ Secured cluster services contain the following components:
 * Scanner (optional)
 * Scanner V4 (optional)
 
+If you use a web proxy or firewall, you must ensure that secured clusters and Central can communicate on HTTPS port 443.
```

Comment:
```suggestion
If you use a web proxy or firewall, you must ensure that Secured Clusters and Central can communicate on HTTPS port 443.
```

I've looked through our repo and half the time we capitalize `Secured Cluster`, the other half we don't so it's quite inconsistent. I do think it should be capitalized as a proper noun though.

Reply:
I think the rule is, if you are talking about secured clusters, as a general type of cluster, it is not capitalized. If you are talking about the name of a specific service, for example, Secured Cluster services, then it is capitalized. This is consistent with guidance in the [IBM](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#nouns) & Red Hat Style Guides. We may have some inconsistencies in old material in the docs, but this is what we try to do in new docs and updates.

Reply:
Oh, we do have a [backlog ticket](https://issues.redhat.com/browse/ROX-23001), but it's a lower priority than other things.